### DISCUSSION TITLE
have not been granted access to the annotation exercise

### DISCUSSION THREAD

**Discussion by gelarehai** [sebastian-hofstaetter/teaching#56] [Advanced IR: Exercise Q&A] [answered]
Hi!

I still have not been granted access to the annotation exercise. Although I sent out my Github info, can you please help me?

Thanks,
Gelareh Taghizadeh

**Comment by sebastian-hofstaetter** [comment, marked as answer]
Hi,
I sent you an email with the information.
Best,
Sebastian

**Comment by gelarehai** [comment]
Thanks very much Sebastian!
